### PR TITLE
MOD gitignore for simulator `dpgh` files

### PR DESCRIPTION
**Pull request by om-ha**
- This issue appears when using simulator to build your app.
- Appeared on `Xcode 12.5.1` & `flutter 2.5.0`

### CODE REVIEW

**Comment by om-ha** [comment]
Two example files:
- `ios/build/Pods.build/Release-iphonesimulator/FirebaseCore.build/dgph`
- `ios/build/Pods.build/Release-iphonesimulator/Flutter.build/dgph`

In fact I think this isn't just about this plugin's pods, all pods are effected by this.

**Comment by om-ha** [comment]
podfile:
```
# Uncomment this line to define a global platform for your project
platform :ios, '13.0'

# CocoaPods analytics sends network stats synchronously affecting flutter build latency.
ENV['COCOAPODS_DISABLE_STATS'] = 'true'

project 'Runner', {
  'Debug' => :debug,
  'Profile' => :release,
  'Release' => :release,
}

def flutter_root
  generated_xcode_build_settings_path = File.expand_path(File.join('..', 'Flutter', 'Generated.xcconfig'), __FILE__)
  unless File.exist?(generated_xcode_build_settings_path)
    raise "#{generated_xcode_build_settings_path} must exist. If you're running pod install manually, make sure flutter pub get is executed first"
  end

  File.foreach(generated_xcode_build_settings_path) do |line|
    matches = line.match(/FLUTTER_ROOT\=(.*)/)
    return matches[1].strip if matches
  end
  raise "FLUTTER_ROOT not found in #{generated_xcode_build_settings_path}. Try deleting Generated.xcconfig, then run flutter pub get"
end

require File.expand_path(File.join('packages', 'flutter_tools', 'bin', 'podhelper'), flutter_root)

flutter_ios_podfile_setup

target 'Runner' do
  use_frameworks!
  use_modular_headers!

  flutter_install_all_ios_pods File.dirname(File.realpath(__FILE__))
end

post_install do |installer|
  installer.pods_project.targets.each do |target|
    flutter_additional_ios_build_settings(target)
  end
end

```

**Comment by bharat-biradar** [comment]
@om-ha Thank you for opening the PR. Please wait until @fbernaly reviews before we can merge it.

**Review by fbernaly — APPROVED** [review]

**Pull request merged**
